src/modules/rules-points/controller/point.controller.js

```diff
@@ -0,0 +1,35 @@
+// RUTA: src/modules/rules-points/controller/point.controller.js
+
+import { getDashboardData, getLeaderboard } from '../service/point.service.js';
+
+/**
+ * Obtiene los datos del dashboard para el usuario actualmente autenticado.
+ */
+export const getDashboardForLoggedUser = async (req, res) => {
+  try {
+    // req.user.id es provisto por el middleware de autenticación
+    const dashboardData = await getDashboardData(req.user.id);
```

Comment:
Antes de llamar getDashboardData(req.user.id), podrías validar que req.user y req.user.id existen. Esto evitaría errores en caso de que el middleware de autenticación falle o no esté presente.

```suggestion
    if (!req.user || !req.user.id) {
        return res.status(401).json({ message: 'Usuario no autenticado.' });
}
```